maintenance_curation.Rmd

```diff
@@ -0,0 +1,238 @@
+    field indicating the date of migration and a URL for a new
+    location if needed.. They will retain "peer-reviewed" tags and
+    badges. They will not be checked via CI.
+
+-   Archived packages will not be displayed by default. An option to
+    "show archived packages" will display these with an "archived"
+    tag.
+
+-   Archived packages may be un-archived at authors' request if authors
+    opt to resume maintenance.
+
+## Legacy acquired packages
+
+"Legacy" packages are packages not created or maintained by rOpenSci
+staff and not peer reviewed, but are under the rOpenSci GitHub
+organization(s) due to historical reasons. (Prior to establishing the
+peer review process and its scope, rOpenSci absorbed packages from
+various developers without well-defined criteria.)
+
+-   rOpenSci will transfer legacy packages back to author organizations
+    and repositories. If authors are uninterested, we will transfer
+    them to the "ropensci-archive" repository. If packages are
+    [[in-scope]{.underline}](https://devguide.ropensci.org/policies.html#aims-and-scope),
+    rOpenSci will inquire if authors would like to submit them to the
+    Software Review process.
+
+-   Legacy packages transferred out to author organizations will not be
+    listed in the package registry.
+
+-   Legacy packages will be tagged "archived" in the package registry
+    if they are in the ropensci-archive repository and unmaintained.
+
+## Incubator packages
+
+"Incubator" packages are in-development packages created by staff or
+community members as part of community projects, such as those created
+at unconferences
+
+-   Incubator packages will live in the "ropensci-incubator"
+    repository (to be renamed from "ropenscilabs")
+
+-   Incubator packages will appear in the package registry with the
+    "incubator" tag
+
+-   Incubator packages will not appear on the website by default, but
+    only if a "include incubator packages" option is checked.
+
+-   Incubator packages will be built and tested via CI (if they have
+    minimal package infrastructure), with documentation clearly
+    labeled as incubator. Authors will not receive alerts for failing
+    packages and status will appear in a different page or section of
+    the testing server.
+
+-   Biannually or annually, rOpenSci will contact incubator maintainers
+    about repositories at least three months old, inquiring about
+    development status and author preferences for migration to
+    peer-review, ropensci-archive, or to author organizations. Based
+    on response, package will be migrated immediately, peer review
+    will be initiated, or migration will be deferred to the next
+    review. Incubator packages will be migrated to ropensci-archive by
+    default after one year.
+
+-   Archived incubator packages will gain the "archived" tag and
+    retain the "incubator" tag in the registry
+
+-   Packages migrated to author organizations will remain in the
```

Comment:
we don't have the infrastructure for this in the registry at the moment.
@sckott how do you think we should keep track of that? A manually updated CSV?
@noamross do we really want to keep track of this vs efforts needed? Genuine question.